scripts/globals/treasure.lua

```diff
@@ -1055,8 +1055,10 @@ xi.treasure.treasureInfo =
                 {
                     {
                         test = function(player)
-                            return (player:getCharVar("Quest[2][77]Prog") == 2 or player:getCharVar("Quest[2][77]Prog") == 3) and
-                            not player:hasKeyItem(xi.ki.JOKER_CARD)
+                            return player:getQuestStatus(xi.quest.log_id.WINDURST, xi.quest.id.windurst.WILD_CARD) == QUEST_ACCEPTED and
```

Comment:
Don't need to check if the quest is accepted. When quest var, Prog, > 0 then the quest will always be accepted, unless it's repeatable. Then it can become more obscure. But when a quest is completed, all vars are uninitialized and deallocated as far as I understand.

Reply:
@claywar has requested on Discord to specifically add quest enums for getQuestStatus to these tests in addition to the quest getVars for the quests progression.

(Technically AF quests are repeatable after a reset through NPC: Vingijard, but I'm not sure if this is implemented yet.)

Reply:
Oh sure thing! I see what he's saying. Sounds good.

Reply:
For posterity, can confirm that char_vars table is cleaned up and the Quest[x][y]Prog entry is gone after fully completing the quest.